modules/edu.gemini.seqexec.server/src/main/resources/Altair.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I removed all Altair channels fro Tcs.xml and put them here. They were causing error messages because Seqexec was attempting to connect to them.